RandomTextFile.txt

```diff
@@ -1,2 +1,4 @@
 This is the text file to test my first branch
-Yay :)
\ No newline at end of file
+Yay :)
+More changes to my wonderful branch
+Frowny face :(
```

Comment:
why are you sad? i think this should be changed to a happy face

Reply:
There are no happy faces without sad ones to compare it to....
Dumbass